about/index.html

```diff
@@ -71,7 +70,6 @@ <h5 class="subheader">{{officer.title}} ({{officer.year}} )</h5>
             {% endif %}
             <ul class="small button-group round right">
               {% if officer.email %}<li><a class="small button" href="mailto:{{officer.email}}">{{officer.email}} <i class="general foundicon-mail"></i></a></li>{% endif %}
-              {% if officer.discourse %}<li><a class="small button" href="{{site.social.forum}}users/{{officer.discourse}}">Get in touch <i class="social foundicon-torso" style="height:51px"></i></a></li>{% endif %}
```

Comment:
Are these links to discourse intended to be removed as part of this pull request? I'm happy to approve the added Docker support, but can't comment on removing discourse links.

Reply:
Pushed in error - have updated the branch